data/src/main/java/com/example/data/implementation/repository/UserRepositoryImpl.kt

```
@@ -321,4 +312,67 @@ class UserRepositoryImpl @Inject constructor(
+            purposes = purposes,
+            interests = interests
+        )
+
+        return try {
+            userApi.completeSocialProfile(
+                authorization = "Bearer $socialToken",
+                body = request
+            )
+            Log.d(TAG, "[소셜 프로필 완료] 성공")
+            true
+        } catch (e: ApiError) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)